definition_validator/validator.cc

```diff
@@ -249,8 +249,10 @@ void validateCompatibleOverride(const core::Context ctx, core::MethodRef superMe
     if (auto leftRest = left.pos.rest) {
         if (!right.pos.rest) {
             if (auto e = ctx.beginError(methodDef.declLoc, core::errors::Resolver::BadMethodOverride)) {
-                e.setHeader("{} method `{}` must accept *`{}`", implementationOf(ctx, superMethod),
-                            superMethod.show(ctx), leftRest->get().show(ctx));
+                auto missingSplatName =
+                    leftRest->get().isAnonymous(ctx) ? "*" : fmt::format("*`{}`", leftRest->get().show(ctx));
+                e.setHeader("{} method `{}` must accept {}", implementationOf(ctx, superMethod), superMethod.show(ctx),
+                            missingSplatName);
```

Comment:
This is weird, but the

```
`{}`
```

4-character string is actually treated specially in error messages. When `--color=auto` or `--color=yes` is enabled, the backticks will get replaced by ANSI control sequences to highlight that region in cyan, instead of the default terminal text color.

So we can't remove the \`{}\` from the `setHeader` call.

Probably better to do something like

```cpp
auto paramName = leftRest->get().isAnonymous(ctx) ? "" : leftRest->get().show(ctx);
```

and leave the *\`{}\` in the setHeader call.

(Applies below too)